Control coverage/Feature/ApplicationGateway.md

```diff
@@ -130,5 +130,109 @@ Enabling DDOS on Vnet of Application Gateway, provides protection and defense fo
 **Properties:** properties.enableDdosProtection
  <br />
 
+<br/>
+ 
+
+## Azure_ApplicationGateway_DP_Enable_EndToEnd_Encryption_Trial
+
+### Display Name 
+[Trial] End to end encryption should be enabled on the Application Gateway.
+
+### Rationale 
+Enabling HTTPS for Application Gateway ensures server/service authentication and protects data in transit from network layer eavesdropping attacks. TLS provides confidentiality and data integrity between client and server. Using approved TLS version significantly reduces risks from security design issues and security bugs that may be present in older versions.
+
+### Control Spec 
+
+> **Passed:** 
+> HTTPS is enabled at Listener settings of the Application Gateway. <b>AND</b>
+> HTTPS is enabled at the backend settings of the Application Gateway.  <b>AND</b>
+> SSL policy if being used, is set to TLS 1.2 or higher.
+> 
+> **Failed:** 
+> HTTPS is not enabled at Listener settings of the Application Gateway. <b>OR</b>
+> HTTPS is not enabled at the backend settings of the Application Gateway. <b>OR</b>
+> SSL policy if being used, is not set to TLS 1.2 or higher.
+> 
+> **Error:** 
+> There was an error fetching Configuration details of Application Gateway.
+> 
+### Recommendation 
+
+- **Azure Portal** 
+
+	 Use the Azure portal to configure end to end encryption on the Application Gateway.
```

Comment:
If we're giving a recomendation then we might want to be specific.

Reply:
And if we can't then might as well remove it.

Reply:
Added detailed recommendation